tests/integration/buildcmd/build_integ_base.py

```diff
@@ -748,7 +741,10 @@ def _verify_built_artifact(self, build_dir, function_logical_id, expected_files)
         self.assertEqual(actual_files, expected_files)
 
     def _get_python_version(self):
-        return "python{}.{}".format(sys.version_info.major, sys.version_info.minor)
+        """
+        This method is used to override python version of some of the tests which is run with Makefile
+        """
+        return "python3.10"
```

Comment:
Does this mean we need to ensure python3.10 is installed on all machines running these tests?

Reply:
This could be any python version that lambda supports today. This was getting system python version before, and it might be causing issues when a version is deprecated by Lambda but still used by SAM CLI or SAM CLI supports a newer version which is not supported by Lambda yet.